CryptoPy/cryptoScraper.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+from requests.exceptions import ConnectionError, Timeout, TooManyRedirects
+
+
+url = 'https://api.nomics.com/v1/currencies/ticker'
+
+
+
+#enter your api key from nomics in "key"
+#curreny symbols: 'BTC' for bitcoin, 'ETH' for ethereum, 'LTC' for litecoin and 'XRP' for ripple
+def getPrice(key, currencySymbol, interval = '1hr'):
+    parameters = {
+        'key': key,
+        'ids': currencySymbol,
+        'interval': interval,
+        'convert': 'USDT'
+    }
+
+    try:
+        response = requests.get(url, params=parameters)
+        # print(response.json())
+        data = response.json()
+        for dp in data:
+            price = dp["price"]
+            # print(dp["symbol"] + ': ' + price)
+        return price
+
```

Comment:
Change the for loop to this -
`price = [dp["price"] for dp in data]
        return(', '.join(price))`